RocketChat/Rocket.Chat
# [FIX] Unable to remove a user who joined a public team with a mention

## Proposed changes (including videos or screenshots)



This PR fixes a bug where a user that joins a team by mention don't get added to the team.



## Issue(s)

Fix #25817

## Steps to test or reproduce

1. Go to the team creation menu. Switch the private setting to off. Create a team.
2. Mention user with @. The user is added.
3. Remove user from the team.
4. The user is not deleted. A Bad Request error appears.

## Further comments
